FreeType.xs

```diff
@@ -407,7 +407,7 @@ qefft2_face (Font_FreeType library, const char *filename, int faceidx, FT_Int32
     PREINIT:
         SV *library_sv;
         QefFT2_Face_Extra *extra;
-        const FT_Bitmap_Size *size;
+        /* const FT_Bitmap_Size *size; */
```

Comment:
No, please, remove. If somebody will need it, he can take on history or send PR where this value is valid.